packages/core/README.md

```diff
@@ -24,7 +24,9 @@ The agent requires a configuration parameter.
 | Property                | Description                                                             | Type                | Optional | Default Value  |
 | ----------------------- | ----------------------------------------------------------------------- | ------------------- | -------- | -------------- |
 | `app`                   | Application metadata                                                    | `App`               | Y        | `undefined`    |
+| `beforeSend`            | Hook to modify or discard an event before it is sent over the wire      | `BeforeSendHook`    | Y        | `undefined`    |
```

Comment:
I'll update description
- [ ] update description

Reply:
reworded description. I'd like to keep event name. It's modeled after other comparable libs, and imho makes sense to name it after event trigger rather than use cases